<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Principal</class>
 <widget class="QMainWindow" name="Principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>818</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Principal</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>400</width>
      <height>721</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>400</width>
      <height>0</height>
     </size>
    </property>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>11</y>
       <width>361</width>
       <height>238</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QWidget" name="widget_5" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Fecha:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="inDia">
           <property name="maximum">
            <number>31</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="inMes">
           <property name="maximum">
            <number>12</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="inAnio">
           <property name="maximum">
            <number>2025</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Peso:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="inPeso"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Altura:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="inAltura"/>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>192</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QWidget" name="widget_4" native="true">
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QPushButton" name="btnCalcular">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(170, 255, 127);</string>
              </property>
              <property name="text">
               <string>Calcular</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnGuardar">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 255, 127);</string>
              </property>
              <property name="text">
               <string>Guardar</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="widget_6" native="true">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>256</y>
       <width>361</width>
       <height>215</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>IMC(kg/m^2):</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="outIMC">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>0.0</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="outCuadro">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 127);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="outDetalleMasa">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="btnGuardarImagen">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 170, 127);</string>
        </property>
        <property name="text">
         <string>Guardar imagen</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="widget_7" native="true">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>478</y>
       <width>321</width>
       <height>221</height>
      </rect>
     </property>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>11</y>
        <width>351</width>
        <height>201</height>
       </rect>
      </property>
      <property name="title">
       <string>Datos</string>
      </property>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>28</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Altura</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>51</y>
         <width>40</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Actual:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>81</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Peso</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>40</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Actual:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Peso maximo:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>78</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Peso minimo:</string>
       </property>
      </widget>
      <widget class="QLabel" name="outPactual">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>110</y>
         <width>55</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>0.0</string>
       </property>
      </widget>
      <widget class="QLabel" name="outPmax">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>140</y>
         <width>55</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>0.0</string>
       </property>
      </widget>
      <widget class="QLabel" name="outPmin">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>170</y>
         <width>55</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>0.0</string>
       </property>
      </widget>
      <widget class="QLabel" name="outAltura">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>50</y>
         <width>55</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>0.0</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionNuevo"/>
    <addaction name="actionGuardar"/>
    <addaction name="actionAbrir"/>
   </widget>
   <addaction name="menuArchivo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNuevo">
   <property name="text">
    <string>Nuevo</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="text">
    <string>Guardar</string>
   </property>
  </action>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
